cfme/tests/control/test_basic.py

```diff
@@ -110,6 +110,18 @@ def random_host_condition():
 
 
 @pytest.yield_fixture
+def random_container_image_condition():
```

Comment:
I would like to remove the name "random" from here and the fixture below. If it is just string that is random, I don't think we should call it a random condition. It implies a level of unrepeatability.